sdf_timing/sdfyacc.py

```diff
@@ -380,21 +380,35 @@ def p_del(p):
     p[0] = p[1]
 
 
-def p_device(p):
-    'device : LPAR DEVICE port_spec real_triple real_triple RPAR'
+def p_delval_list(p):
+    '''delval_list : real_triple
+                   | real_triple real_triple
+                   | real_triple real_triple real_triple'''
+
+    # TODO: There should be more possibilities up to 12 real_triples
+    # But we stick with 1, 2 or 3 for now.
+
     paths = dict()
-    paths['fast'] = p[4]
-    paths['slow'] = p[5]
-    device = utils.add_device(p[3], paths)
+    if len(p) == 2:
+        paths['nominal'] = p[1]
+    elif len(p) == 3:
+        paths['fast'] = p[1]
+        paths['slow'] = p[2]
+    elif len(p) == 4:
+        paths['fast'] = p[1]
+        paths['nominal'] = p[2]
+        paths['slow'] = p[3]
+    p[0] = paths
+
+def p_device(p):
+    'device : LPAR DEVICE port_spec delval_list RPAR'
```

Comment:
These should all be triple quoted strings, not single quotes.

Reply:
Fixed.